### ISSUE TITLE
Add ability to remove KMS Key Encryption from LogGroups

### ISSUE THREAD

**Issue by phas13** [Han-Lon/terraform-aws-cloudwatch-logs-management#2]
In case we want to remove KMS key encryption from Log Groups, it does not work. It looks like Lambda function does not manage it. Something like those should be added to log_management.py:

```
        if os.environ.get("KMS_KEY_ALIAS", "None") == "None":
            try:
                cw.disassociate_kms_key(
                    logGroupName=log_group['logGroupName'],
                )
            except Exception as e:
                print(f"Ran into error when encrypting log group {log_group['logGroupName']} in {cw.meta.region_name}")
                print(traceback.format_exc())
```

**Comment by Han-Lon** [comment]
Hi @phas13, thanks for bringing this to my attention and apologies for the late reply. I'll take a look at this tomorrow in my own dev environment and see if that fix works

**Comment by Han-Lon** [comment]
Resolved in the 0.2.0 release after testing in my own environment. Please reopen this issue if it persists on your end.
